packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -125,6 +126,58 @@ class CupertinoScrollbar extends StatefulWidget {
   /// {@endtemplate}
   final ScrollController controller;
 
+  /// {@template flutter.cupertino.cupertinoScrollbar.displayAlways}
+  /// Indicates whether the [Scrollbar] should always be visible.
+  ///
+  /// When false, the scrollbar will be shown during scrolling and will fade out otherwise.
```

Comment:
Make sure to use a line break if any of these comments are longer than 80 characters.